pkg/scheduler/operations.go

```diff
@@ -35,6 +35,28 @@ type OperationsRegistry interface {
 	SetFailed(correlationID, schedulingID, reason string) error
 }
 
+type OperationNowFoundError struct {
```

Comment:
Should be `OperationNotFoundError` :)

Reply:
haha...  but Now would actually also be right! :D Let me fix this typo